packages/flutter_tools/test/general.shard/resident_web_runner_test.dart

```diff
@@ -130,7 +130,10 @@ void main() {
     when(mockFlutterDevice.devFS).thenReturn(mockWebDevFS);
     when(mockFlutterDevice.device).thenReturn(mockDevice);
     when(mockWebDevFS.connect(any)).thenAnswer((Invocation invocation) async {
-      return ConnectionResult(mockAppConnection, mockDebugConnection);
+      return ConnectionResult(
+        mockAppConnection,
+        mockDebugConnection,
+        mockDebugConnection.vmService);
```

Comment:
Nit: at a trailing `,` after `vmService` and put `);` on a newline.

Reply:
done, thanks!